package-overrides/npm/angular-aria@1.5.0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "format": "global",
+  "jspmNodeConversion": false,
+  "main": "angular-aria",
+  "dependencies": {
+    "angular": "npm:angular@^1.5.0"
```

Comment:
Yes the point is that the whole ecosystem has to agree on a main source for angular, otherwise we end up with Angular being duplicated which should not happen.